addons/l10n_es/data/account_chart_template_data.xml

```diff
@@ -39,5 +39,33 @@
         <field name="transfer_account_code_prefix">572999</field>
         <field name="parent_id" ref="account_chart_template_common"/>
     </record>
+    <record id="account_chart_template_coop_common" model="account.chart.template">
+        <field name="name">PGCE Cooperativas común</field>
```

Comment:
If we want to continue with the same structure as the other COA, "Común" should be written with capital letter. Shouldn't it?